amundsen_application/static/js/ducks/bookmarks/api/v0.ts

```diff
@@ -0,0 +1,39 @@
+import axios, { AxiosResponse, AxiosError } from 'axios';
+
+
+// Add or remove bookmarks for the logged in user
+export function addBookmark(action) {
```

Comment:
Would we not also pass in the user's id? Bookmarking involves two resources, not sure how the metadata service is supposed to know which user to add or remove the bookmark from without the user id.

Reply:
For actions that involve updating the logged-in user's data, I think it's safer to add the userId from the authentication layer. Otherwise a user could arbitrarily add or remove bookmarks for another user.